### PR TITLE
admin can create, read, update, delete user

### PR DESCRIPTION
**Pull request by MichelleCCodes**
## Description
- admin can create, read, delete, update user
Fixes # (issue)
- note: errors don't sync with Okta. This means that changes can be seen on your local but may not reflect what's on Okta db. Will need to look into this more.

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] No duplicate code left within changed files

### CODE REVIEW

**Comment by MichelleCCodes** [comment]
Not sure why package-lock json was altered.... Might be unsafe to merge?

**Review by sebastian-espeset — DISMISSED** [review]
Peeped the package-lock.json and it looks like "lockFileVersion" changed from 1 to 2. I don't know whether it's safe to merge or not. Looking into it.

**Review by WeiWu2 — APPROVED** [review]
The package lock changes seems to be caused by a new version of npm being released I just did a npm install and my package lock changed to verison 2 as well so these changes should be fine

**Review by nwestallen — APPROVED** [review]
Data model returns and endpoint responses are more consistent and readable now and fields are renamed correctly to work with db schema

File: api/profile/profileModel.js [hunk 1]
```diff
@@ -56,16 +56,14 @@ const findById = async (user_id) => {
   return user;
 };
 
-const findPendingById = async (user_id) => {
-  const user = await db('pending_profiles')
-    .where({ user_id })
-    .first()
-    .select('*');
+const findPendingById = async (id) => {
```
Comment (dgamboa) [inline api/profile/profileModel.js:59]: This one is different than the one on line 15, right?

File: api/profile/profileModel.js [hunk 2]
```diff
@@ -81,7 +79,8 @@ const update = (user_id, profile) => {
 };
 
 const remove = async (user_id) => {
-  return await db('profiles').where({ user_id }).del();
+  await db('profiles').where({ user_id: user_id }).del();
+  return await db('profiles');
```
Comment (dgamboa) [inline api/profile/profileModel.js:83]: Something the project might consider in the future is to get the user that is being deleted, save it to a variable, delete it the way we do here, then return the user so the router can return a success response with the user deleted

**Review by dgamboa — APPROVED** [review]

File: api/profile/profileModel.js (hunk 1 above)
Comment (MichelleCCodes) [reply api/profile/profileModel.js:59]: Actually, line 15 is obsolete. I'll make a note to clean that up!

File: api/profile/profileModel.js (hunk 2 above)
Comment (MichelleCCodes) [reply api/profile/profileModel.js:83]: Thanks for making note of that! There is an alert that tells the admin which user had just been deleted as you suggested, and then resets the component's state with the new list of profiles that's being returned here. We got you!

**Pull request merged**